swagger/api/generic/severity.yaml

```diff
@@ -0,0 +1,9 @@
+severity:
+  title: Severity
+  description: This is the over all severity or the severity of the most significant item.
+  enum:
+    - indetermined
```

Comment:
Hmzz, do we really want to allow for undetermined? If so it needs to be rectified.